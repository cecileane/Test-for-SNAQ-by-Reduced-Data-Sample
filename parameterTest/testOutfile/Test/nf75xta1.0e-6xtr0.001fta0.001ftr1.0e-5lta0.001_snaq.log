optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 75, liktolAbs = 0.001.
rootname for files: nf75xta1.0e-6xtr0.001fta0.001ftr1.0e-5lta0.001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 381
seed: 381 for run 1
Sat Oct  1 16:58:52 2016

 BEGIN SNaQ for run 1, seed 381 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 133 (failures: 75) with -loglik=215.7344 and ht_min=[0.42251,0.11872,0.0,2.2113,1.00295]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 31		 31		 2	 |	 0		 0		 --
mvtarget	 36		 36		 1	 |	 0		 0		 --
chdir		 9		 9		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 56		 28		 0	 |	 2		 2		 --
Total		 133		 105		 4	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 215.73440080554295
((1,((5,#H7:::0.42250841586683535):2.2112997490040316,6):1.002952378395609):0.11871771946278452,(2,4):0.0,(3)#H7:::0.5774915841331647);
---------------------
seed: 91214 for run 2
Sat Oct  1 16:59:02 2016

 BEGIN SNaQ for run 2, seed 91214 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 98 (failures: 75) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 0	 |	 0		 0		 --
mvtarget	 28		 28		 1	 |	 0		 0		 --
chdir		 9		 9		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 32		 25		 0	 |	 5		 5		 --
Total		 98		 91		 4	 |	 5		 5		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 2, -loglik of best 155.49625734559459
((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7:::0.626975454241492):2.1558602715370543):2.1558602715370543,4,#H7:::0.373024545758508);
---------------------

Sat Oct  1 16:59:07 2016
MaxNet is ((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7:::0.626975454241492):2.1558602715370543):2.1558602715370543,4,#H7:::0.373024545758508); 
with -loglik 155.49625734559459
