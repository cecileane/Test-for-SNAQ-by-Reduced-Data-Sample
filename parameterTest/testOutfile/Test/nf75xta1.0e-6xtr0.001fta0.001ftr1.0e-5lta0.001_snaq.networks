((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7:::0.626975454241492):2.1558602715370543):2.1558602715370543,4,#H7:::0.373024545758508);, with -loglik 155.49625734559459 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((5,6):0.326466334183043,3):0.0,(4)#H7:::0.6784860566243078,(1,(#H7:::0.3215139433756922,2):4.3104941587052075):1.1045510360461082);, with -loglik 158.8652080996225
((((5,6):0.3228411771963024,3):0.0,#H7:::0.4814661969674383):0.5704170745029445,4,((1)#H7:::0.5185338030325617,2):9.73621327186663);, with -loglik 170.68293225821287
((((5,6):0.4194646493403748,3):0.0)#H7:2.092979059037695::0.6244933917625874,4,((1,#H7:1.9571797202205832::0.3755066082374126):1.2907003407443185,2):9.53383423161371);, with -loglik 178.68373292995065
