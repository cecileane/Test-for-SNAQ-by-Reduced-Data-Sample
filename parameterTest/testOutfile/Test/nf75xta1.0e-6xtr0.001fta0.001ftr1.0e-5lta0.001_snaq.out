((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7:::0.626975454241492):2.1558602715370543):2.1558602715370543,4,#H7:::0.373024545758508); -Ploglik = 155.49625734559459
 Dendroscope: ((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7):2.1558602715370543):2.1558602715370543,4,#H7);
 Elapsed time: 14.716434443 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((((5,6):0.33534775534376016,3):0.12573282262678323,(1,(2)#H7:::0.626975454241492):2.1558602715370543):2.1558602715370543,4,#H7:::0.373024545758508);, with -loglik 155.49625734559459
 ((1,((5,#H7:::0.42250841586683535):2.2112997490040316,6):1.002952378395609):0.11871771946278452,(2,4):0.0,(3)#H7:::0.5774915841331647);, with -loglik 215.73440080554295
-------
